layers/meta-balena-5x-owa/recipes-core/system_maintenance/system-maintenance_1.00.bb

```diff
@@ -0,0 +1,23 @@
+DESCRIPTION = "Binary for system_maintenance.sh"
+SECTION = "Binaries"
+DEPENDS = ""
+LICENSE = "CLOSED"
+FILESEXTRAPATHS:prepend := "${THISDIR}/src:"
+
+SRC_URI = " file://system_maintenance.sh "
+
+S = "${WORKDIR}"
+RDEPENDS:${PN} += "bash"
+
+INSANE_SKIP:${PN} += "already-stripped"
+
+do_install() {
+    # create the /usr/bin folder in the rootfs with default permissions
+    install -d ${D}${bindir}
+    install -m 0755 -d ${D}${sysconfdir}
+
+    # install the application into the /usr/bin folder with default permissions
+    install ${WORKDIR}/system_maintenance.sh ${D}${bindir}
+}
+
+RDEPENDS:system-maintenace ?= "bash"
```

Comment:
This `system-maintenance` application should be part of the containerized application.

Reply:
Moved to [PR 37](https://github.com/balena-os/balena-owa5x/pull/37)